zappa/handler.py

```diff
@@ -593,19 +621,39 @@ def handler(self, event, context):
                             ).decode("utf-8")
                             zappa_returndict["isBase64Encoded"] = True
                         else:
-                            zappa_returndict["body"] = response.get_data(as_text=True)
+
+                            gzipped_body, gzipped_content_length = _get_gzipped_body(response)
```

Comment:
It would be neat if you would verify that the `headers["Accept-Encoding"]` contains "gzip" before doing the compression. This would make this feature work according to the current http specs. See

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding

Reply:
Or maybe use https://github.com/colour-science/flask-compress instead of  implementing it your self?